src/Text/LLVM/AST.hs

```diff
@@ -1061,19 +1061,51 @@ data BitOp
 
 -- | Conversions from one type to another.
 data ConvOp
-  = Trunc
-  | ZExt
+  = Trunc Bool Bool
+    -- ^ Truncate an integer value to a smaller integer type.
+    --
+    -- The 'Bool' fields (added in in LLVM 20) encode whether to perform
+    -- overflow-related checks:
+    --
+    -- * First 'Bool': check for unsigned overflow.
+    -- * Second 'Bool': check for signed overflow.
```

Comment:
Do these added bools have the same semantics as https://github.com/GaloisInc/llvm-pretty/pull/168/files#diff-c60254e336a88f05888286b397c8ba49f578b01ce4ce3cff4a1d6431008a89e2R1652-R1675?  I wonder if those flags should be extracted and generalized to use in both locations?

I'll note that the two flags seem mutually exclusive to me (both here and in the reference location above), and it bothers me that they could thus both be set to true.  I think this may be an artifact of how they need to be referenced in the bitcode file but I also wonder if you agree about their exclusivity and if we should thus use a trinary value here instead of two individual and independent flags?